### PR TITLE
Hamburger menu for smaller screens

### PR DESCRIPTION
**Pull request by asia-t**
Resolves https://github.com/ertrzyiks/yummy/issues/30

### CODE REVIEW

File: src/components/navbar/navbar.module.sass [hunk 1]
```diff
@@ -13,9 +25,49 @@
...
+
+.navbar_link_expanded
+  display: inline-block
+  background-color: #ffc16e
+  border-bottom: 2px solid #ffdeb3
+  padding: 14px 0
+  text-align: center
+  text-shadow: none
+
+.navbar_link_expanded:last-child
```
Comment (ertrzyiks) [inline src/components/navbar/navbar.module.sass]:
```
&:last-child
```

File: src/components/navbar/navbar.module.sass [hunk 2]
```diff
@@ -13,9 +25,49 @@
...
+    border-bottom: none
+
+.navbar_link_expanded:hover, .navbar_link_expanded:active
```
Comment (ertrzyiks) [inline src/components/navbar/navbar.module.sass]:
```
&:hover,
&:active
```

File: src/components/menu_close.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+/**
+ * This icon comes from Material Design icons by Google
+ * https://material.io/tools/icons/
+ */
+const Icon = (options) => (
+  <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" {...options}>
```
Comment (ertrzyiks) [inline src/components/menu_close.js]:
```suggestion
  <svg xmlns="http://www.w3.org/2000/svg" width="30" height="30" viewBox="0 0 24 24" {...options}>
```

File: src/components/navbar/index.js [hunk 4]
```diff
@@ -1,29 +1,84 @@
...
-     render={data => (
-       data.allRecipeCategory.edges.map(({node: {slug, name}}) => (
-         <Link to={`/${slug}`} key={slug} className={styles.navbar_link}>{titleize(name)}</Link>
-       ))
-     )} />
-  </nav>
+      `}
+       render={data => (
+         data.allRecipeCategory.edges.map(({node: {slug, name}}) => (
+           <Link to={`/${slug}`} key={slug} className={linkStyle}>
```
Comment (ertrzyiks) [inline src/components/navbar/index.js]: Click on link should close the menu

File: src/components/navbar/index.js [hunk 5]
```diff
@@ -1,29 +1,84 @@
...
+          </span>
+        </a>
+        <a className={[styles.navbar_menu_icon, this.state.menuOpen ? '' : styles.hidden].join(' ')}
+           aria-label="Close category menu"
+           onClick={this.handleMenuCloseClick}>
+          <span aria-hidden="true">
+            <MenuCloseIcon className={styles.icon_menu} />
+          </span>
+        </a>
+        <StaticQuery query={graphql`
```
Comment (ertrzyiks) [inline src/components/navbar/index.js]: Container grouping all the nav links would simplify the styling

**Comment by ertrzyiks** [comment]
screenshot 2019-02-28 18 54 34

Clickable area around icons is too big

File: src/components/header/header.module.sass [hunk 6]
```diff
@@ -19,6 +19,9 @@
   margin-top: 12px
   margin-bottom: 22px
 
+  +media('<tablet')
+    margin-top: 69px
```
Comment (ertrzyiks) [inline src/components/header/header.module.sass]: consider moving it to the navbar itself

File: src/components/navbar/index.js [hunk 7]
```diff
@@ -1,29 +1,84 @@
...
+      : styles.navbar_link
+
+    const overlay = this.state.menuOpen ? (
+        <div className={styles.navbar_menu_overlay} onClick={this.handleMenuCloseClick} />)
+      : null;
+
+    return (
+      <nav className={styles.navbar}>
+        {overlay}
+        <a className={[styles.navbar_menu_icon, this.state.menuOpen ? styles.hidden : ''].join(' ')}
```
Comment (ertrzyiks) [inline src/components/navbar/index.js]: No href = no tabbing, `href="#"` would do the trick

**Comment by asia-t** [comment]
@ertrzyiks I've made the changes - would you care to retest?

**Pull request merged**